snooty/util.py

```diff
@@ -324,13 +324,21 @@ def get(
             datetime.timedelta(hours=1) if cache_interval is None else cache_interval
         )
 
+        url_netloc = urllib.parse.urlparse(url).netloc
+        is_raw_gh_content_url = url_netloc == "raw.githubusercontent.com"
+        target_url = (
+            f"https://deploy-preview-1310--docs-frontend-stg.netlify.app/.netlify/functions/fetch-url?url={url}"
```

Comment:
Reminder for myself to switch this out with the deployed URL for this function instead of the deploy preview